templates/repo/commits_table.tmpl

```diff
@@ -61,6 +61,10 @@
 						</td>
 						<td class="message collapsing">
 							<span class="has-emoji{{if gt .ParentCount 1}} grey text{{end}}">{{RenderCommitMessage .Summary $.RepoLink $.Repository.ComposeMetas}}</span>
+							{{if IsMultilineCommitMessage .Message}}
+							<button class="basic compact mini ui icon button commit-button"><i class="ellipsis horizontal icon"></i></button>
+							<pre class="commit-body" style="display: none;">{{RenderCommitBody .Message $.RepoLink $.Repository.ComposeMetas}}</pre>
```

Comment:
There is inline CSS here, could you change it out with a CSS class?
Also this requires JS which means if someone has JS disabled then they won't be able to see the long commit message (perhaps default to open when JS is disabled?)

Reply:
Or perhaps hide the button if the user doesn't have JS is another option

Reply:
@techknowlogick I think for this inline style is ok

Reply:
I agree it's not the prettiest, but the `toggle` function changes the style inline so I thought it was a decent compromise.